modules/pulsar/src/test/java/org/testcontainers/containers/PulsarContainerTest.java

```diff
@@ -23,6 +23,13 @@ public void testUsage() throws Exception {
         }
     }
 
+    @Test
+    public void shouldWaitForFunctionsWorkerStarted() {
```

Comment:
Is there anything we can assert to ensure that it actually works?
Otherwise, this test will pass if I make `withFunctionsWorker` NO-OP 😅

Reply:
log output changes, so its more like a check for a waiter. I can check that log output really contains worker output, but that would duplicate the waiter. Uploading function would require admin dependency. I can do that too and just commit really simple jar with rudimentary function

Reply:
implemented with real function

Reply:
Isn't there some flag to check or something?
If not, let's use `java.util.Objects#requireNonNull` as a function, to avoid having to have a jar file